Update tutorial-kubernetes-prepare-acr.md

mikloscari opened the pull request:
updated the reference to point to "az acr list" link, as well following the convention on the links internal for point the right site.

azurecla commented:
Hi __@neumanndaniel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Byronr commented:
@neumanndaniel Thanks for the update. @ Tysonn Please merge.

neumanndaniel commented:
Still open and not merged.

mikloscari committed:
Update tutorial-kubernetes-prepare-acr.md

updated the reference to point to "az acr list" link, as well following the convention on the links internal for point the right site.
